charts/tembo-ai/values.yaml

```diff
@@ -0,0 +1,144 @@
+inferenceGateway:
+  image:
+    repository: quay.io/tembo/inference-gateway
+    pullPolicy: IfNotPresent
+    tag: latest
+  resources:
+    requests:
+      cpu: 100m
+      memory: 410Mi
+    limits:
+      cpu: 1000m
+      memory: 512Mi
+  livenessProbe:
+    enabled: true
+    path: /lively
+    port: http
+  readinessProbe:
+    enabled: true
+    path: /ready
+    port: http
+  autoscaling:
+    enabled: true
+    minReplicas: 1
+    maxReplicas: 10
+    # Target CPU is a percentage of requests, i.e. the measurement can
+    # exceed 100% when limits is greater than requests.
+    targetCPUUtilizationPercentage: 99
+  externalSecrets:
+    refreshInterval: "5m"
+    parameterStore:
+      name: "secret-store-parameter-store"
+      kind: ClusterSecretStore
+    secretName: ~
+    secretRegex: ~
+  podMonitor:
+    enabled: false
+    path: /metrics
+    # Sometimes applications serve metrics on a different port,
+    # which makes it easier to prevent metrics from accidentally
+    # being publicly available.
+    portName: metrics
+    containerPort: 8081
+  ingressRoute:
+    enabled: false
+    # match: "Host(`example.com`)"
+    match: ~
+  serviceAccount:
+    create: true
+    annotations: {}
+  service:
+    port: 8080
+  args: []
+  command:
+    - "/usr/local/bin/gateway"
+  env: []
```

Comment:
can we assign env vars from secrets here? `DATABASE_URL` will need to be set on the gateway server.

Reply:
It might be nice to add envs like this https://github.com/tembo-io/app-deploy-dev/blob/main/dev/tembo-ai.yaml

since then it's easier to just swap envs when we need. And this is subject to the list merging issue in yaml, i.e. can't merge in defaults from values file, rather the entire list needs defined if any value overwritten in the app deploy.

Also, I thought it is assuming the path in secrets store, so I feel that should be an added config instead of a default perhaps

```
            - name: DATABASE_URL
              valueFrom:
                secretKeyRef:
                  name: data-plane-inference
                  key: _data-plane_inference_db_connection_string
```

Reply:
I see, ok that works for me. I thought this was replacing the private app-deploy process, my mistake.